### ISSUE TITLE
docker-compose installation does not work

### ISSUE THREAD

**Issue by dmr** [pinry/pinry#120]
I ran the installation as explained in the README:

`docker-compose up`
`docker-compose exec web python3 manage.py migrate --settings=pinry.settings.docker`

Everything worked nicely but now I don't see the static files:

![image](https://user-images.githubusercontent.com/128286/36348551-6474a3d6-1472-11e8-800e-98b406dfabd7.png)

Did I miss something?

**Comment by dmr** [comment]
User registration worked without statics :)

**Comment by overshard** [comment]
Yep, it's something I'm working on at the moment, I'm unsure how to handle static files with this setup. I'll leave this open so we don't get additional questions about it and maybe someone can help fix it. 😄

**Comment by dmr** [comment]
I really like your docker-compose setup idea!
In other projects we used Whitenoise (http://whitenoise.evans.io/en/stable/) to solve the static files serving issues and it works great.
It's probably not as fast as nginx but fast enough and easy to setup (just another middleware).

**Comment by overshard** [comment]
That's what I use with https://github.com/overshard/timestrap however the problem here is media static files that the user uploads. Whitenoise doesn't handle that.
